src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -452,7 +452,7 @@ static bool IsApplicableBase(const ConvolutionContext& params)
         return false;
 
     const auto name = params.GetStream().GetDeviceName();
-    if(!(StartsWith(name, "gfx9") || StartsWith(name, "gfx10")))
+    if(!(StartsWith(name, "gfx9") || StartsWith(name, "gfx10")) || name == "gfx90a")
```

Comment:
This should be legalized as `WORKAROUND_ISSUE_1093`. Please see #561.

Reply:
[Note] In fact, this PR moves W/A from tests to the library.

Reply:
@atamazov While I make gfx90a test disable as default in ctest, it still raised error from test_conv_2d compilation. So I wrote this line in library.